### PR TITLE
Web Interface for Scoper

### PR DESCRIPTION
**Pull request by nityanandagohain**
hey @RameshAditya I have created a web interface for scoper.

I have used Flask for developing it.

Here is the demo.
demoscoper

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        
+        # check if the values are present
+        if not (youtube_link and algorithm and limit):
+            return render_template("index.html", error="please enter proper values in the form", empty=True)
+                
+        # get the query results
+        results = FINDER.main(youtube_link, query=query, limit=5, languages=["en"], mode=algorithm)
+
+        # converting results from [('apple watch.', '29m 56s')] to [['apple watch.', '29m56s']]
+        print("results before", results)
```
Comment (RameshAditya) [inline app.py]: This seems like a debug message. I'd suggest either write these to a separate log file or remove this line. :)

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+<body>
+    <nav>
+        <div class="nav-wrapper">
+            <a href="/" class="brand-logo" style="margin-left: 4rem;">Scoper</a>
+            <ul id="nav-mobile" class="right hide-on-med-and-down">
+                <li><a href="https://github.com/" target="_blank"><i class="fa fa-github fa-2x" aria-hidden="true"></i></a></li>
+            </ul>
+        </div>
+    </nav>
+
```
Comment (RameshAditya) [inline templates/index.html]: Could we add a line break over here, to separate the nav bar from the main content of the webpage? :)

File: scoper.py [hunk 3]
```diff
@@ -334,7 +331,8 @@ def main(self, youtube_video_url, limit=5, languages=None, mode='FUZZY'):
         for timestamp in timestamps:
             captions_extracted.append(timestamps_and_captions[timestamp])
 
-        pretty_print(list(zip(captions_extracted, normalize_time(timestamps))))
+        # pretty_print(list(zip(captions_extracted, normalize_time(timestamps))))
```
Comment (RameshAditya) [inline scoper.py]: Also, I think it's better if to uncomment this line as commenting it mutes the CLI output, so if someone were not using the web GUI, they wouldn't be seeing any output.

**Comment by RameshAditya** [comment]
Hi @nityanandagohain, nice work - I've suggested a few changes, let me know if they look good to you and if so I can merge the updated code.

**Comment by nityanandagohain** [comment]
Hey, @RameshAditya  I have noted it. I was also wondering what should be done to the readme. Also, l guess we should keep both the CLI and the web version. Please let me know what do you think?

**Comment by RameshAditya** [comment]
The changes look good, I'll merge them and for the readme, I plan on adding instructions on using the web GUI under the [usage section](https://github.com/RameshAditya/scoper#how-to-use-scoper). :) 

Ultimately the user will have full flexibility between using the CLI or the GUI versions.

**Pull request merged**